app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -124,7 +130,22 @@ public void displayDescriptionActivityData() {
         getParticipantNames();
         loadActivityHeaderPicture();
 
-        findViewById(R.id.activityGPSRecDescription).setEnabled(activity.getSport() == Sport.Running);
+        View recButton = findViewById(R.id.activityGPSRecDescription);
+        if (activity.getSport() == Sport.Running) {
+            recButton.setVisibility(View.VISIBLE);
```

Comment:
Ok nice, so I guess, as planned the "GPS track recording" feature will be available only for running activities. Otherwise the button is set as `INVISIBLE`, is that correct?

Reply:
Indeed! I was afraid of using `GONE` since it might have messed up @jlstoupy98 's layouts